src/sizing/BodyMeasurements.py

```diff
@@ -0,0 +1,76 @@
+from numpy import mean
+
+from src.sizing.women_size import measurement_to_size, size_to_measurement
+
+
+class BodyMeasurements:
```

Comment:
Should we have this as a dataclass?
e.g.
```python
@dataclass
class BodyMeasurements:
    size: int | None
    waist: float | None
    # etc
```
size could then be a property that returns either the initialised size or estimates from given measurements.
```python
@property
def size(self):
    if self.size is None:
        return self.estimate_size()
    return self.size
```